components/Wizard/formComponents/Packages.js

```diff
@@ -0,0 +1,537 @@
+import React, { useState, useRef, useEffect, useCallback } from "react";
+import useFormApi from "@data-driven-forms/react-form-renderer/use-form-api";
+import useFieldApi from "@data-driven-forms/react-form-renderer/use-field-api";
+import PropTypes from "prop-types";
+import {
+  Bullseye,
+  Button,
+  DualListSelector,
+  DualListSelectorPane,
+  DualListSelectorList,
+  DualListSelectorListItem,
+  DualListSelectorControlsWrapper,
+  DualListSelectorControl,
+  SearchInput,
+  Spinner,
+  TextContent,
+} from "@patternfly/react-core";
+import { AngleDoubleLeftIcon, AngleLeftIcon, AngleDoubleRightIcon, AngleRightIcon } from "@patternfly/react-icons";
+import { connect } from "react-redux";
+import api from "../../../core/api";
+import * as composer from "../../../core/composer";
+import { updateBlueprintComponents } from "../../../core/actions/blueprints";
+
+// the fields isHidden and isSelected should not be included in the package list sent for image creation
+const removePackagesDisplayFields = (packages) =>
+  packages.map((pack) => ({
+    name: pack.name,
+    summary: pack.summary,
+  }));
```

Comment:
This is a helper specifically for the Packages component so can you include it inside of the Packages function.

Reply:
Will do!

Reply:
In fact, this is actually no longer necessary now that we set the packages form field to the list of package names and can just be removed.